recipes/finagg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "finagg" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/finagg-{{ version }}.tar.gz
+  sha256: fc3b7071283fd55af835a7fb3098e1f3c28b9fc880f98196ed4342173a008ce2
+
+build:
+  entry_points:
+    - finagg = finagg.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=64.0
+    - setuptools-scm >=5
+    - pip
+  run:
+    - python >=3.10
+    - beautifulsoup4 ==4.*
+    - click ==8.*
+    - pandas ==1.*
+    - pandas-stubs ==1.*
+    - python-dotenv ==0.21.*
+    - requests ==2.*
+    - requests-cache ==0.9.*
+    - sqlalchemy ==2.*
+    - tqdm ==4.*
+    - types-requests ==2.*
+    - typing-extensions ==4.*
```

Comment:
I believe these tight  pins are upstream, right? This is a recipe for disaster. While we can merge this as the upstream authors intended, we should probably report it and ask for a more relaxed pinnning.